content/en/hub/tasks/advanced/system-dataset.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Changing the System Dataset"
+description: "A how-to on changing the system dataset."
+---
+
+The system dataset stores debugging core files, encryption keys for encrypted
+pools, and Samba4 metadata such as the user/group cache and share level
+permissions.
+
+
+
+## Change System Dataset
+
+To view the current location of the system dataset, go to
+**System > System Dataset**. The system dataset can be changed by selecting an
+existing pool from the *System Dataset Pool* dropdown. The system dataset can
+be moved to unencrypted pools or encrypted pools which do not have
+passphrases. If the system dataset is moved to an encrypted pool, that
+volume is no longer allowed to be locked or have a passphrase set.
```

Comment:
Has this been rechecked for the new ZFS encryption model? This might only apply to the previous GELI implementation.

Reply:
Good point. Forgot about that. I'll test it.

Reply:
@Mrt134 I've tested. It still applies, except now you cannot switch the system dataset to a pool with an existing passphrase at all. Text has been updated to reflect.